admin/app/components/badges/badge.hbs

```diff
@@ -31,30 +31,57 @@
 
 
     <div class="page-section__details table-admin">
-      <table>
-        <thead>
-          <tr>
-            <th>Périmètre</th>
-            <th>Seuil</th>
-          </tr>
-        </thead>
-        <tbody>
-          {{#each @badge.badgeCriteria as |criterion|}}
-            <tr>
-              <td>{{criterion.scope}}
-                {{#if criterion.partnerCompetences}}
-                  <ul>
-                    {{#each criterion.partnerCompetences as |badgePartnerCompetence|}}
-                      <li>{{badgePartnerCompetence.name}}</li>
+      {{#each this.badgeCriteria as |criterion|}}
+        {{#if (eq criterion.scope "SomePartnerCompetences")}}
+          <p>L'évalué doit obtenir <strong>{{criterion.threshold}}%</strong> sur l'un des groupes de compétences suivants :</p>
+        {{/if}}
+        {{#if (eq criterion.scope "EveryPartnerCompetences")}}
+          <p>L'évalué doit obtenir <strong>{{criterion.threshold}}%</strong> sur l'ensemble des groupes de compétences suivants :</p>
+        {{/if}}
+        {{#if (eq criterion.scope "CampaignParticipation")}}
+          <p>L'évalué doit obtenir <strong>{{criterion.threshold}}%</strong> sur l'ensemble des compétences du target profile.</p>
+        {{/if}}
```

Comment:
proposition de simplification : 
```suggestion
          <p>L'évalué doit obtenir <strong>{{criterion.threshold}}%</strong> sur {{fn this.scopeExplaination criterion.scope}}</p>
```

Dans le js :
```js
get scopeExplaination(criterionScope) {
	switch (criterionScope) {
		case "SomePartnerCompetences": return "l'un des groupes de compétences suivants :";
		case "EveryPartnerCompetences": return "l'ensemble des groupes de compétences suivants :";
		case "CampaignParticipation": return "l'ensemble des compétences du target profile.";
	}
}
```

Reply:
`{{fn this.scopeExplaination criterion.scope}}` Ah c'est le pattern qu'on cherchait hier !